src/teams/templates/emails/add_membership_email.html

```diff
@@ -5,6 +5,24 @@
 Thank you for helping out!
 <br>
 <br>
+<h3>Team contact info</h3>
+<h5>IRC Channel</h5>
+{% if team.public_irc_channel_name %}
+  <p>The {{ team.name }} Team public IRC channel is <a href="irc://{{ IRCBOT_SERVER_HOSTNAME }}/{{ team.public_irc_channel_name }}">{{ team.public_irc_channel_name }} on {{ IRCBOT_SERVER_HOSTNAME }}</a>.
+{% endif %}
+{% if request.user in team.approved_members.all and team.private_irc_channel_name %}
```

Comment:
this should be a permission check for `team.member_permission_set` instead of a membership check